src/index.html

```diff
@@ -11,7 +11,147 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="form-fieldset">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            minlength="2"
+            maxlength="20"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="name"
+            minlength="2"
+            maxlength="20"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input type="number"
+          name="age"
+          min="1"
+          max="100"
+          value="12">
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input type="date"
+          name="date-of-birth">
+        </label>
+
+        <label>
+          I accept the term of the agreement
+          <input type="checkbox"
+          name="terms"
+          required>
+        </label>
+      </fieldset>
+
+      <fieldset class="form-fieldset">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input type="email"
+          name="E-mail"
+          placeholder="email@example.com">
```

Comment:
```suggestion
          <input
            type="email"
            name="E-mail"
            placeholder="email@example.com"
          >
```
follow the [style guide](https://mate-academy.github.io/style-guides/htmlcss.html) through out your code 😉